eventtracking/caching.py

```diff
@@ -0,0 +1,103 @@
+"""
+Utilities for managing cache.
+"""
+from inspect import signature
+from logging import getLogger
+
+from edx_django_utils.cache import TieredCache, get_cache_key
+
+
+logger = getLogger(__name__)
+
+
+def cached(cls):
+    """
+    Decorator for a class to help the caching methods differentiating
+    between standalone mathods and class/instance methods.
+    """
+    cls.is_cached_class = True
+    return cls
+
+
+def cache_value(default_key_params=None, verbose_name='', get_key_params=None):
```

Comment:
we should not have our own abstraction of caching utils it should be part of django-utils.

Reply:
Removed